client/views/chat.tpl

```diff
@@ -33,4 +35,4 @@
 		</div>
 	</aside>
 </div>
-{{/each}}
+{{/each}}
```

Comment:
Yeah, this comment.

I thought this would break the build, but I guess this PR was added before travis was set up. Is there a way to nudge travis that this PR exists?